src/components/map/toolbar/toolbarModeButtons.tsx

```diff
@@ -2,12 +2,41 @@ import React from 'react';
 import classnames from 'classnames';
 import EditMode from '~/enums/editMode';
 import toolbarStore from '~/stores/toolbarStore';
+import routeBuilder from '~/routing/routeBuilder';
+import subSites from '~/routing/subSites';
+import navigator from '~/routing/navigator';
 import { RadioButton } from '../../controls';
 import * as s from './toolbarModeButtons.scss';
 
 export default class ToolbarModeButtons extends React.Component {
-    private toggleSelectedMode = (option: EditMode) => () => {
-        toolbarStore.setEditMode(option);
+    constructor(props: any) {
+        super(props);
+
+        this.initStores();
+    }
+
+    private initStores() {
+        if (navigator.getPathName() === subSites.network) {
+            toolbarStore.setEditMode(EditMode.NETWORK);
+        }
```

Comment:
We could move the initial set of EditMode into this function, and leave the initial EditMode as undefined. Otherwise, with this change, we have two locations where we initially set the mode. Both in this function and in the store.

Reply:
Removed init from store. Didn't want to initialize it as undefined (would have to add type EditMode|undefined) and undefined is not a valid value for it